src/icons/knowledgeUserInfo.svg

```diff
@@ -0,0 +1,72 @@
+<svg width="360" height="264" viewBox="0 0 360 264" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think that the class="icon" should be added instead of having a set width and height